Mage/src/main/java/mage/Mana.java

```diff
@@ -1053,11 +1166,11 @@ public boolean getFlag() {
      */
     public void setToMana(final Mana mana) {
         this.any = mana.any;
-        this.red = mana.red;
-        this.green = mana.green;
         this.white = mana.white;
```

Comment:
Must be copy operation here. 

P.S. If any cards uses direct set operation for mana objects outside from that class then it will be big fail like a static filters bug in old days. Must be checked.

Reply:
damn, thought I did that